src/commands/utility/search-my-points.js

```diff
@@ -0,0 +1,77 @@
+const { SlashCommandBuilder } = require("discord.js");
+require("dotenv").config(); 
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("my-points-query")
+    .setDescription(
+      "Get the Discord search query to check points for a specific month"
+    )
+
+    .addIntegerOption((option) =>
+      option.setName("year").setDescription("Year to search").setRequired(false)
+    )
+
+    .addIntegerOption((option) =>
+      option
+        .setName("month")
+        .setDescription("Month to search")
+        .setRequired(false)
+    )
+
+    .addUserOption((option) =>
+      option
+        .setName("user")
+        .setDescription("User to search for")
+        .setRequired(false)
+    )
+
+    .addStringOption((option) =>
+      option
+        .setName("channel")
+        .setDescription("Select a specific channel")
+        .setRequired(false)
+        .addChoices(
+          { name: "novabot", value: "novabot" },
+          { name: "code-review", value: "code-review" }
+        )
+    ),
+
```

Comment:
if possible allow user select various

Reply:
I did it this way
![image](https://github.com/user-attachments/assets/d38a470f-7b9c-424f-8462-d125bd148d82)